vegafusion-python/pyproject.toml

```
@@ -15,17 +14,25 @@ classifiers = [
   "License :: OSI Approved :: BSD License",
   "Topic :: Scientific/Engineering :: Visualization",
 ]
-dependencies = ["arro3-core", "packaging", "narwhals>=1.42"]
+dependencies = [
+  "arro3-core",
+  "packaging",
+  "narwhals>=1.42",
+  "datafusion>=48.0.0,<49.0.0",
```

there is a version mistmach between python and rust since latter is using datafusion 49.0.0 (in Cargo.toml line 104)
```suggestion
  "datafusion>=49.0.0,<50.0.0",
```